schematic/store/synapse.py

```diff
@@ -100,23 +101,46 @@ def __init__(
 
         self._query_fileview()
 
+    def _purge_synapse_cache(self):
+        '''
+        Purge synapse cache if it exceeds 19.5 GB
+        '''
+        # try clearing the cache
+        # scan a directory and check size of files
+        root_dir = self.syn.cache.cache_root_dir
+        cache = self.syn.cache
+        if os.path.exists(root_dir):
+            nbytes = get_dir_size(root_dir)
+            # if 19.5 GB has already been taken, purge cache before an hour
+            if nbytes >= 20937965568:
+                an_hour_earlier = datetime.strftime(datetime.utcnow()- timedelta(hours = 1), '%s')
+                num_of_deleted_files = cache.purge(before_date = int(an_hour_earlier))
+                print(f'{num_of_deleted_files} number of files have been deleted from {root_dir}')
```

Comment:
Here should I use a simple print statement or logger? I think print statement might be better since I could see it on AWS

Reply:
hmm - a google search revealed this: https://stackoverflow.com/questions/37703609/using-python-logging-with-aws-lambda.  

If the rest of the package uses `logging`, I would use `logging` as well.

Reply:
@thomasyu888 thanks for the link. I have updated the code to use logger/logging